packages/runner/src/lib/state.js

```diff
@@ -1,5 +1,6 @@
 import { action, computed, observable } from 'mobx'
-import { automation, BaseStore } from '@packages/runner-shared'
+import { MobxRunnerStore } from '@packages/app/src/store/mobx-runner-store'
```

Comment:
Does this create a circular dependency?

Reply:
I realized this file is also not used, so I'm just going to delete it. Can't have a circular dep without this file :100:

Reply:
https://github.com/cypress-io/cypress/pull/20663/commits/02f75439e3fb8dfb640e254aa9b4d1f505f23f29